css/style.css

```diff
@@ -9,400 +14,633 @@ body {
 }
 
 .instagram {
-    margin-right: 15px;
+  margin-right: 15px;
 }
 
 .vk {
-    margin-right: 15px;
+  margin-right: 15px;
 }
 
 .fb {
-    margin-right: 500px;
+  margin-right: 500px;
 }
 
 .line {
-    position: absolute;
-    top: 12px;
-    right: 12px;
+  position: absolute;
+  top: 12px;
+  right: 12px;
 }
 .img {
-    position: relative;
+  position: absolute;
+  top: 3%;
+  right: 0;
 }
+
 .bag__title-bold {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 40px;
-    line-height: 41px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
 }
 
 .bag__title {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 16px;
-    line-height: 16px;
-    letter-spacing: .07em;
-    opacity: .5;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: .07em;
+  opacity: .5;
 }
 
 .bag-text{
-    font-style: normal;
-    font-weight: normal;
-    font-size: 15px;
-    line-height: 21px;
-    opacity: .5;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  opacity: .5;
 }
 
 .product {
-    font-style: normal;
-    font-weight: 500;
-    font-size: 13px;
-    line-height: 16px; 
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px; 
 }
 
 .bag-title-bold {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 40px;
-    line-height: 41px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
 }
 
 .bag-title {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 16px;
-    line-height: 16px;
-    letter-spacing: .07em;
-    opacity: .5;  
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: .07em;
+  opacity: .5;  
 }
 
 .total-look__text {
-    font-style: normal;
-    font-weight: normal;
-    font-size: 15px;
-    line-height: 21px;
-    opacity: .5;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  opacity: .5;
 }
 
 .name {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 12px;
-    line-height: 15px;
-}
-
-.new-collection__background {
-    background-size: 100% auto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 15px;
 }
 
+/* Cтили 2й секции ниже */
 
-/* Медиа запросы */
+.bag {
+  background: #FAE8E4;
+  height: 900px;
+  margin-top: 0;
+  width: 1440px;
+}
 
-@media (max-width: 1270px) { 
-   .title {
-       top: 30%;
-       width: 20%;
-       font-size: 50px;
-       line-height: 52px;
-       left: 60%;
-   }
-    } 
+.bag__wight {
+  position: absolute;
+  width: 398px;
+  height: 415px;
+  margin-left: 137px;
+  top: 1117px;
+}
 
-@media (max-width: 1270px) { 
-    .text {
-        width: 35%;
-        left: 60%;
-        height: 60%;
-    }
+.bag__pink {
+  position: absolute;
+  width: 582px;
+  height: 844px;
+  margin-left: 231px;
+  top: 928px; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)